packages/shared/src/utils/slugify/builtinReplacements.ts

```diff
@@ -0,0 +1,2065 @@
+/* cspell:disable */
+
+export const builtinReplacements = [
```

Comment:
How much does this add to the bundle size of @novu/shared? Since a lot of our libs still depend on it, is it maybe safer to add the slugify code to application-generic?

Reply:
`slugify` is needed by the frontend, so without a `@novu/utils` distinct from a `@novu/types`, this is the only way at the moment

Reply:
@SokratisVidros I'm going to merge this to get the fix in ahead of tomorrow's bug bashing session. I won't release `shared` yet in case we need to take a different direction for home of `slugify`.